Resources/Textures/Clothing/Uniforms/Jumpsuit/ancient.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at commit (Will update when PR'd), monkey made by brainfood1183 (github) for ss14",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/blob/662c08272acd7be79531550919f56f846726eabb/icons/mob/uniform.dmi, monkey made by brainfood1183 (github) for ss14",
```

This link can't be a blob, this needs to be an explicit commit.

```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/662c08272acd7be79531550919f56f846726eabb/icons/mob/uniform.dmi, monkey made by brainfood1183 (github) for ss14",
```